### PR TITLE
use __ENABLE_TESTS_GLES to on/off egl compilation

### PR DESCRIPTION
**Pull request by LindaYu17**
Signed-off-by: Linda Yu linda.yu@intel.com

### CODE REVIEW

File: tests/egl/gles2_help.c [hunk 1]
```diff
@@ -23,6 +23,8 @@
 #include "config.h"
 #endif
 
+#if __ENABLE_TESTS_GLES__
+
```
Comment (chivakker) [inline tests/egl/gles2_help.c]: This check should be moved out from this file in the same way it was removed from the libyami patches.

**Comment by LindaYu17** [comment]
thanks for the comments, already updated.

File: tests/v4l2decode.cpp [hunk 2]
```diff
@@ -519,9 +521,11 @@ bool takeOneOutputFrame(int fd, int index = -1/* if index is not -1, simple enqu
 #elif __ENABLE_WAYLAND__
         ret = displayOneVideoFrameWayland(fd, &buf);
 #else
-        if (IS_DMA_BUF() || IS_DRM_NAME())
+        if (IS_DMA_BUF() || IS_DRM_NAME()) {
+#ifdef __ENABLE_TESTS_GLES__
             ret = displayOneVideoFrameEGL(fd, buf.index);
-        else
+#endif
+        } else
```
Comment (chivakker) [inline tests/v4l2decode.cpp]: when __ENABLE_TESTS_GLES__ is not defined the whole if{} else will be a no operation.  The #ifdef has to be located outside the if (IS_DMA_BUF()...)

File: tests/v4l2decode.cpp [hunk 3]
```diff
@@ -848,6 +852,7 @@ int main(int argc, char** argv)
             }
         }
     } else if (IS_DMA_BUF() || IS_DRM_NAME()) {
+#if __ENABLE_TESTS_GLES__
```
Comment (chivakker) [inline tests/v4l2decode.cpp]: same thing here, the else if has to be placed within the __ENABLE_TEST_GLES__ compilation flag.

**Comment by LindaYu17** [comment]
done

**Comment by xuguangxin** [comment]
good to me, thanks Linda and Daniel. We will merge this after 1.0.1 released.

**Comment by xuguangxin** [comment]
meged as 2e5dc8395fed2e720794aea7750f2f542abfeb14, thanks Daniel and Linda

**Pull request closed without merging**